tests/__sf_section_dir.test.fish

```diff
@@ -4,11 +4,20 @@ function setup
 	spacefish_test_setup
 	mkdir -p ~/.tmp-spacefish/dir1/dir2
 	mkdir -p /tmp/tmp-spacefish/dir1/dir2/dir3
```

Comment:
To slim down on the setup in each test to do with write protected directories, I'd suggest adding this to the setup so you can just `cd` into them at the start of the test:
```suggestion
	mkdir -p /tmp/tmp-spacefish/dir1/dir2/dir3
	mkdir -p /tmp/tmp-spacefish/writeProtected
	chmod 500 /tmp/tmp-spacefish/writeProtected
```